common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -165,7 +165,7 @@ export default function SendAssetsForm({
                       accountAsset => accountAsset.uuid === values.asset.uuid
                     ) || { balance: '0' }
                   ).balance
-                : getBalanceFromAccount(values.account);
+                : formatEther(getBalanceFromAccount(values.account));
```

Comment:
I like the idea of custom wrappers around ethers methods. We could start using them in future work.